src/ui/layouts/Delegation.tsx

```diff
@@ -176,7 +175,8 @@ export const DelegateSection = ({ delegates }: { delegates: Delegate[] }) => {
 };
 
 export function DelegationPanel({ state }: { state: State }) {
-  const { delegates } = state;
+  // ToDo: read delegates from state
```

Comment:
It should be removed?

Reply:
I set it to read from mock data, for now, to be able to debug the UX. This is just a reminder to switch back to state when it is ready!

Reply:
It seems the state includes the delegates. I removed the mock delegates.